app/models/conversation.rb

```diff
@@ -83,7 +83,7 @@ def ordered_participants
   end
 
   def subject
-    self[:subject].blank? ? "no subject" : self[:subject]
+    self[:subject].blank? ? I18n.t('conversations.new.subject_default') : self[:subject]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.